src/gennylib/include/gennylib/context.hpp

```diff
@@ -443,8 +443,10 @@ class PhaseContext final : public V1::ConfigNode {
 
         // Check to make sure we haven't broken our rules
         if (isNop && _node.size() > 1) {
-            throw InvalidConfigurationException(
-                "Nop cannot be used with any other keywords. Check YML configuration.");
+            if (_node.size() != 2 || !_node["Phase"]) {
```

Comment:
I have dreams of a whitelist comparison for this. (I.E. I'm a NoOp so I only allow Phase, NoOp, Operation, and Operations at the root level.) That said, I think it's too much effort.